contracts/src/v0.8/functions/dev/v1_X/FunctionsBilling.sol

```diff
@@ -124,6 +142,27 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     return SafeCast.toUint96((1e18 * amountWei) / getWeiPerUnitLink());
   }
 
+  /// @inheritdoc IFunctionsBilling
+  function getUsdPerUnitLink() public view returns (uint256, uint8) {
+    FunctionsBillingConfig memory config = s_config;
```

Comment:
What is the reason for creating this new memory variable here?
Can't we just access `s_config` directly?

Reply:
Huh.. just tested this, you're right. Glad you questioned that.